axolotl-web/src/pages/DeviceList.vue

```diff
@@ -31,7 +27,7 @@ export default {
   props: {
     msg: String
   },
-  created(){
+  mounted(){
```

Comment:
The fix for the list not showing up is here.

Reply:
I wonder, why it was working at all :), I think some race condition between my fast desktop and slower phones